client/app/views/steps/agreement.coffee

```diff
@@ -14,12 +14,17 @@ module.exports = class AgreementView extends StepView
 
     onRender: ->
         @$errorContainer=@$('.errors')
+        @$statsPart = @$('.stats-agreement')
 
         if @error
             @renderError(@error)
         else
             @$errorContainer.hide()
 
+        # Environment variable to hide stats checkbox part
+        if ENV.HIDESTATS
```

Comment:
I am not a big fan of referring to ENV in views. Maybe the agreement view should have a `hideStats()` method called from `Application`, like it has been done here https://github.com/cozy/cozy-proxy/blob/development/client/app/application.coffee#L120